server/Student.ts

```diff
@@ -43,14 +43,17 @@ class Student extends EventEmitter {
                 case "send_reaction":
                     this.handlerSendReaction(parsed);
                     break;
+                case "kill_me":
```

Comment:
Maybe would be better  `delete`?

Reply:
![image](https://user-images.githubusercontent.com/44035398/117550889-18c82780-b043-11eb-8e2b-c5b389876def.png)